### PR TITLE
chore: upgrade APISIX to 3.3.0

### PR DESCRIPTION
**Pull request by Gallardot**

### CODE REVIEW

**Review by leslie-tsang — APPROVED** [review]

**Comment by AlinsRan** [comment]
Change the default router from radixtree_uri to radixtree_host_uri.
Perhaps we should be consistent with APISIX?
 
https://github.com/apache/apisix-helm-chart/blob/5092739f02dd9d29587ed319f797f23e0992c019/charts/apisix/values.yaml#L65-L69

https://github.com/apache/apisix/blob/master/CHANGELOG.md#change

**Comment by Gallardot** [comment]
> Change the default router from `radixtree_uri` to `radixtree_host_uri`. Perhaps we should be consistent with APISIX?
> 
> https://github.com/apache/apisix-helm-chart/blob/5092739f02dd9d29587ed319f797f23e0992c019/charts/apisix/values.yaml#L65-L69
> 
> https://github.com/apache/apisix/blob/master/CHANGELOG.md#change

Done. PTAL. @AlinsRan

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .idea/
+.vscode/
 .DS_Store
 test_**.yaml
 .history
```
Comment (leslie-tsang) [inline .gitignore:5]: Need a new line at the end of the file.

**Review by AlinsRan — APPROVED** [review]

**Comment by tao12345666333** [comment]
I'm not certain if there have been any other configuration changes. Could you please make a check?

**Comment by Gallardot** [comment]
> I'm not certain if there have been any other configuration changes. Could you please make a check?

Sure, after inspecting the [changelog documentation](https://github.com/apache/apisix/blob/master/CHANGELOG.md#330) shows that [#9047](https://github.com/apache/apisix/pull/9047) is the only breaking change that involves a default configuration item.

**Review by tao12345666333 — APPROVED** [review]
thanks

**Pull request merged**